src/doc/src/reference/workspaces.md

```diff
@@ -110,7 +110,7 @@ is not inside a subdirectory of the workspace root.
 
 #### Package selection
 
-In a workspace, package-related cargo commands like [`cargo build`] can use
+In a workspace, package-related Cargo commands like [`cargo build`] can use
```

Comment:
I think this maybe is right. Because in Cargo's document. We use Cargo to indicate the Cargo project. And use cargo to indicate the command or the binary. So probably here we just mean `cargo` commands.

Reply:
You got the point. That's the pattern I am aware of. However, looking into the surrounding context, I feel like the change might be right, as it doesn't refer to any specific command.